docs/getting_started.md

```diff
@@ -0,0 +1,67 @@
+# Getting Started
+
+kssh allows you to define realms of servers where access is granted based off of
+membership in different teams. Imagine that you have a staging environment that everyone should be granted access to and
+a production environment that you want to restrict access to a smaller group of people. For this exercise we'll also set
+up a third realm that grants root access to all machines. 
+
+Start by creating a new Keybase user to use for the CA chatbot:
+
+```bash
+keybase signup      # Creates a new Keybase user to use for the SSH CA bot
+keybase paperkey    # Generate a new paper key
+```
+
+Note that this system will not work if you attempt to use the same user for the CA chatbot as for kssh. It is required
+to use a unique user. 
+
+Then create `{TEAM}.ssh.staging`, `{TEAM}.ssh.production`, `{TEAM}.ssh.root_everywhere` as new Keybase subteams
+and add the bot to those subteams. Add users to those subteams based off of the permissions you wish to grant
+different users
+
+On a secured server that you wish to use to run the CA chatbot:
```

Comment:
the only thing that needs to be on this server is docker, yes? that might be worth mentioning right here.

Reply:
Yup! Added a comment